templates/service_page.hbs

```diff
@@ -14,13 +14,18 @@
   const settings = {{json settings}};
   const container = document.getElementById("service-catalog-item");
 
+  // Extract the ID from the URL
+  const url = window.location.href;
+  const id = url.substring(url.lastIndexOf('/') + 1);
+
   const props = {
     baseLocale: {{json help_center.base_locale}},
     hasAtMentions: {{json help_center.at_mentions_enabled}},
     userRole: {{json user.role}},
     userId: {{json user.id}},
     brandId: {{json brand.id}},
     organizations: {{json user.organizations}},
```

Comment:
Nit as it's not related to this PR but we might not be using all these properties in the app.

Reply:
When we will be sure what type of fields we will have we can revise this.